### PR TITLE
[DOC] Add tedana development setup instructions

### LINKED ISSUE #192: Developer Tedana Config: import failure
When trying to run tedana without 
```pip install /path/to/tedana```
(which seems an inelegant solution), and instead using
```python tedana.py [opts]```
the following error message triggers
```python ./tedana/workflows/tedana.py
Traceback (most recent call last):
 File "./tedana/workflows/tedana.py", line 15, in <module>
   from tedana import decay, combine, decomposition, io, model, selection, utils
 File "/home/lod23/code/tedana/tedana/workflows/tedana.py", line 15, in <module>
   from tedana import decay, combine, decomposition, io, model, selection, utils
ImportError: cannot import name 'decay' from 'tedana' (/[tedana_path]/tedana/tedana/workflows/tedana.py)
```
I'm not sure if there's something I'm missing in the documentation or if the intended developer run method is to pip install. I think it would be good to be able to simply call python on the cloned version, though, if possible, and include documentation on how to do so correctly if that functionality exists.

### PR DESCRIPTION
**Pull request by jbteves**
Closes #192 .

Changes proposed in this pull request:

- In "Making a Change," adds instructions to run setup.py for developers

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -111,6 +110,12 @@ Try to keep the changes focused. We've found that working on a [new branch][link
 
 When you're creating your pull request, please make sure to review the tedana [style conventions](#style-guide).
 
+Note that to test a change, you may need to set up your local repository to run a tedana workflow. To do so, run
+```
+python setup.py develop
+```
+which should ensure all packages are correctly organized and linked on your user profile. Once you've run this, your repository should be set for most changes (i.e., you do not have to re-run with every change).
```
Comment (emdupre) [inline CONTRIBUTING.md]: Sorry to be nitpicky, but could you put each sentence on a new line

Comment (jbteves) [reply CONTRIBUTING.md]: That's fine, if you aren't nitpicky code style can get out of hand quickly, I understand.

**Comment by jbteves** [comment]
I think this latest commit should address that, let me know if anything else needs to be changed.

**Comment by emdupre** [comment]
LGTM. Thx !

**Pull request merged**